string mode
float32[] targetPose
